### PR TITLE
Ignore ci for some subdirectories and files

### PR DESCRIPTION
**Pull request by jneira**

### CODE REVIEW

File: .github/workflows/bench.yml [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
+    branches:
+      - '**'
+    paths-ignore:
+      - '**/docs/**'
+      - '**.md'
+      - '**/LICENSE'
+      - '.circleci/**'
+      - 'install/**'
+      - 'nix/**'
+      - 'test/**'
```
Comment (jneira) [inline .github/workflows/bench.yml:14]: I think bench are not related with tests in any way, right?

File: .github/workflows/bench.yml [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+  pull_request:
+    branches:
+      - '**'
+    paths-ignore:
+      - '**/docs/**'
+      - '**.md'
+      - '**/LICENSE'
+      - '.circleci/**'
+      - 'install/**'
+      - 'nix/**'
```
Comment (pepeiborra) [inline .github/workflows/bench.yml:13]: nix shouldn't be ignored

**Review by pepeiborra — APPROVED** [review]

**Review by Ailrun — APPROVED** [review]
Awesome, thank you for the update!

File: .github/workflows/bench.yml (hunk 1 above)
Comment (pepeiborra) [inline .github/workflows/bench.yml:14]:
```suggestion
      - 'test/**'
      - 'ghcide/test/**'
```

**Pull request merged**